settings/com.endlessm.settings.gschema.override.in

```diff
@@ -152,3 +152,6 @@ enabled=true
 [org.gnome.desktop.peripherals.touchpad]
 click-method='default'
 tap-to-click=true
+
+[org.gnome.ControlCenter]
```

Comment:
If `--disable-faces` is specified, then this setting shouldn't be specified. But honestly, implementing that would not be worth the trouble. Instead, I wouldn't bother with the enable option, I can't think of a use case for it.

Reply:
Done, I've removed the enable option from the configure.ac